scripts/release_sdk_status/main.py

```diff
@@ -313,15 +313,15 @@ def read_file(file_name):
 def find_test_path(line: str) -> str:
     line = line.strip('\n') + '\n'
     try:
-        return re.findall('output-folder: \$\(python-sdks-folder\)/(.*?)\n', line)[0]
+        return re.findall('output-folder: \\$\\(python-sdks-folder\\)/(.*?)\n', line)[0]
```

Comment:
```suggestion
        return re.findall(r'output-folder: \$\(python-sdks-folder\)/(.*?)\n', line)[0]
```

Reply:
Just need to add `r` to avoid warning.

Reply:
but there is a LF (`'\n'`) at the end, `r` makes it backslash `"'"` + `'n'`